gitbook/features/parallel.md

```
@@ -19,7 +19,12 @@ Parallel mode is off by default. Enable it in preferences to use `/gsd parallel`
    ```
    /gsd parallel start
    ```
-   GSD scans milestones, checks dependencies and file overlap, shows an eligibility report, and spawns workers.
+   GSD scans milestones, checks dependencies and file overlap, shows an eligibility report, reconciles shared state, and spawns workers.
+
+   If reconciliation cannot repair current drift, no workers are launched and the command prints:
+   ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)